test/board_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/cell'
+require './lib/ship'
+require '.lib/board'
+
+class BoardTest < Minitest::Test
+
+  def test_instance_of_board
+
+    assert_instnace_of board = Board.new
+
+  end
+
+  def test_board_has_cells_in_hash_type
+    board = Board.new
+    board.cells
+
+    assert_equal Hash, board.class
+  end
+
+  def test_validating_coordinates
+    board = Board.new
+
+    assert_equal true, board.valid_coordinate?("A1")
+    assert_equal true, board.valid_coordinate?("D4")
```

Comment:
I was thinking we use :symbols for the coordinate name since they are something we wont be changing over time. Thoughts?

Reply:
If you check out iteration 2, i'm matching what they gave us as the test/return.